pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
Yes, it does. Actually I am not sure we should set any default for this. @smarterclayton how do you envision defaulting here?

cc: @pweil- @ramr @liggitt